src/style/index.css

```
@@ -361,8 +364,18 @@
       overflow: visible;
     }
 
-    /* Collapse all other content when this is present */
-    body:has(#single-print) *:not(#single-print):not(#single-print *) {
+    /* Collapse everything else for single-print (only when multi-print is NOT also active) */
+    body:has(#single-print):not(:has(#multi-print)) *:not(#single-print):not(#single-print *) {
```

Stylelint (17.3.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)